roles/kubernetes/preinstall/tasks/main.yml

```diff
@@ -140,14 +140,10 @@
   tags: bootstrap-os
 
 - name: Enable ip forwarding
-  lineinfile:
-    dest: /etc/sysctl.d/99-sysctl.conf
-    regexp: '^net.ipv4.ip_forward='
-    line: 'net.ipv4.ip_forward=1'
+  sysctl:
+    name: net.ipv4.ip_forward
```

Comment:
the issue is that GCE overwrites that via cloud init, and only properly ordered sysctl conf files can do the magic. If that works, you could try like this: 
* check if the 99th is a symlink, and if it is - use another file name, f.e. indexed as 990
* otherwise use the file as it is